docs/source/test_network.md

```diff
@@ -0,0 +1,585 @@
+./network.sh createChannel
+```
+If the command was successful, you can see the following message printed in your
+logs:
+```
+========= Channel successfully joined ===========
+```
+
+You can also use the channel flag to create a channel with custom name. As an
+example, the following command would create a channel named `channel1`:
+```
+./network.sh createChannel -c channel1
+```
+
+The channel flag also allows you to create multiple channels by specifying
+different channel names. After you create `mychannel` or `channel1`, you can use
+the command below to create a second channel named `channel2`:
+```
+./network.sh createChannel -c channel2
+```
+
+If you want to bring up the network and create a channel in a single step, you
+can use the `up` and `createChannel` modes together:
+```
+./network.sh up createChannel
+```
+
+## Starting a chaincode on the channel
+
+After you have created a channel, you can start using [smart contracts](smartcontract/smartcontract.html) to
+interact with the channel ledger. Smart contracts contain the business logic
+that governs assets on the blockchain ledger. Applications run by members of the
+network can invoke smart contracts to create assets on the ledger, as well as
+change and transfer those assets. Applications also query smart contracts to
+read data on the ledger.
+
+To ensure that transactions are valid, transactions created using smart contracts
+typically need to be signed by multiple organizations to be committed to the
+channel ledger. Multiple signatures are integral to the trust model of Fabric.
+Requiring multiple endorsements for a transaction prevents one organization on
+a channel from tampering with the ledger on their peer or using business logic
+that was not agreed to. To sign a transaction, each organization needs to invoke
+and execute the smart contract on their peer, which then signs the output of the
+transaction. If the output is consistent and has been signed by enough
+organizations, the transaction can be committed to the ledger. The policy that
+specifies the set organizations on the channel that need to execute the smart
+contract is referred to as the endorsement policy, which is set for each
+chaincode as part of the chaincode definition.
+
+In Fabric, smart contracts are deployed on the network in packages referred to
+as chaincode. A Chaincode is installed on the peers of an organization and then
+deployed to a channel, where it can then be used to endorse transactions and
+interact with the blockchain ledger. Before a chaincode can be deployed to a
+channel, the members of the channel need to agree on a chaincode definition that
+establishes chaincode governance. When the required number of organizations
+agree, the chaincode definition can be committed to the channel, and the
+chaincode is ready to be used.
+
+After you have used the `network.sh` to create a channel, you can start a
+chaincode on the channel using the following command:
+```
+network.sh installCC
```

Comment:
This should be deployCC. I don't see installCC in network.sh script.

Reply:
done.